share/jupyter/hub/static/less/login.less

```diff
@@ -27,7 +29,65 @@
         margin-top: -1px;
     }
     
-    button[type=submit] {
+    input[type=submit] {
         margin-top: 16px;
+        background-color: @login-primary;
+        background: linear-gradient(to bottom,  @login-primary 0%, @login-secondary 100%); /* W3C */
```

Comment:
can you get rid of the gradients? They don't fit in well with the otherwise flat colors.